pandas_gbq/gbq.py

```diff
@@ -733,6 +733,7 @@ def read_gbq(
     project_id=None,
     index_col=None,
     col_order=None,
+    columns=None,
```

Comment:
I don't think it matters too much here, but technically this is a breaking change. Python allows these arguments to be passed in positionally or by keyword unless we use some special syntax to force them to use it as a keyword argument.